proposals/2021-09-hub-ibc-router/README.md

```diff
@@ -0,0 +1,25 @@
+# ⚛️ Make the Cosmos Hub the IBC Router ⚛️
+
+The following is a selection from the [Cosmos Whitepaper](https://v1.cosmos.network/resources/whitepaper):
+
+```
+The Cosmos Hub connects to many other blockchains (or zones) via a novel inter-blockchain communication protocol. The Cosmos Hub tracks numerous token types and keeps record of the total number of tokens in each connected zone. Tokens can be transferred from one zone to another securely and quickly without the need for a liquid exchange between zones, because all inter-zone coin transfers go through the Cosmos Hub.
+
+...
+
+Any of the zones can themselves be hubs to form an acyclic graph, but for the sake of clarity we will only describe the simple configuration where there is only one hub, and many non-hub zones.
+```
+
+The Hub has long been envisioned as a central point in the IBC architecture. In the battle to build and ship IBC this central vision has remained unchanged, but with so much focus on the need to build out other zones with real economies to support this network (the CosmosSDK is the result of this effort), the idea of the hub as an Interchain Router hasn't been discussed in a serious context for quite a while.
+
+This is understandable: Cosmos needed so many other pieces to come together before the Hub had a chance to even start performing this function. Those other zones have been created, they each have products and economies. The bootstrapping era of IBC is well underway. 
+
+These new zones joining are noticing a problem: they need to maintain a large amount of infrastructure (archive nodes and relayers for each counterparty chain) to connect with all the chains in the ecosystem, a number that is continuing to increase quickly.
+
+Luckly this problem has been anticipated and IBC architected to accomodate multi-hop transactions. However, a packet forwarding/routing feature was not in the initial IBC release. This proposal aims to fix this for the Hub.
+
+Strangelove Ventures has delivered an [IBC Middleware module](https://github.com/cosmos/ibc-go/pull/373) that will allow the hub to play the role of IBC Router that was always envisioned for it. Passing of this propsal will begin the era of the Hub offering interchain services to other chains and profiting from those relationships.
+
+To pay the hub validators and stakers, this proposal implements a governance configurable fee (which we propose should be initially set to 0.0 to encourage adoption) that will be taken out of each packet and given to the community pool. The community pool will then periodically trade these fees for ATOM and distribute them to staked holders. The exact distribution method of these fees is left TBD in this proposal as it is not initially required and can be implemented in a future governance proposal. One way to do this would be using the [Groups module](https://docs.cosmos.network/master/architecture/adr-042-group-module.html), Community spend proposals and the Gravity DEX.
```

Comment:
I think it's good to include this as a general incentive for ATOM holders to benefit from the hub as a router (beyond more transaction fees in general). However I could imagine a scenario where the hub is competing with another hub-like zone who offers 0 fees. Appreciate that by starting at 0 the hub can scale its offering with the needs of the market. My only point being that if adding fees this way makes the work significantly more difficult and less likely to ship quickly, I could imagine justifying removing it as a premature optimization.

Reply:
The fee collection is in the implementation right now. I think that once hub becomes router will be hard for other zones to compete because of the number of connections and quality of service the hub can provide.